### PR TITLE
Update FAQ: SDK roadmap

### PR DESCRIPTION
**Pull request by gennidou**

### CODE REVIEW

**Comment by gennidou** [comment]
Adding back the link to the roadmap for when navigation on the docs page is fixed

**Review by NPJ-OP-LUX — APPROVED** [review]
Looks fine.

File: modules/ROOT/pages/FAQ/index.adoc [hunk 1]
```diff
@@ -741,7 +741,10 @@ For more information and examples of metadata driven applications: https://docs.
 
 Is there a roadmap (release plan) for future eForms SDK releases or a set release date for SDK versions?::
 
-The eForms SDK is a complicated development and information is made available as fast as possible. 
+The eForms SDK is a complicated development and information is made available as fast as possible.
+The SDK roadmap is available at link:https://docs.ted.europa.eu/eforms-common/roadmap/index.html[eForms SDK roadmap]
```
Comment (bertrand-lorentz) [inline modules/ROOT/pages/FAQ/index.adoc:745]: We should not put the full URL for links to other pages in the docs.
See https://docs.antora.org/antora/latest/page/xref/

**Review by NPJ-OP-LUX — APPROVED** [review]
Looks fine

**Pull request closed without merging**